src/plotrecipes.jl

```diff
@@ -0,0 +1,105 @@
+shapecoords(geom::AbstractPoint) = [tuple(coordinates(geom)...)]
+RecipesBase.@recipe f(geom::AbstractPoint) = (seriestype --> :scatter; legend --> :false; shapecoords(geom))
```

Comment:
There should also be `aspect_ratio := 1` in all recipes, which forces units on the x and y axes to be the same – otherwise the projection will be lost.